tests/mutaterows_test.go

```diff
@@ -85,7 +86,7 @@ func dummyMutateRowsRequestCore(tableID string, rowKeys []string) *btpb.MutateRo
 func dummyMutateRowsRequest(tableID string, numRows int) *btpb.MutateRowsRequest {
 	rowKeys := []string{}
 	for i := 0; i < numRows; i++ {
-		rowKeys = append(rowKeys, "row-" + strconv.Itoa(i))
+		rowKeys = append(rowKeys, "row-"+strconv.Itoa(i))
```

Comment:
nit: why all the format changes?

Reply:
This is best practice. I have run [gofmt](https://go.dev/blog/gofmt) on the modified files.